iotdb-core/datanode/src/main/java/org/apache/iotdb/db/schemaengine/schemaregion/mtree/traverser/Traverser.java

```diff
@@ -73,6 +74,9 @@ public abstract class Traverser<R, N extends IMNode<N>> extends AbstractTreeVisi
   // default false means fullPath pattern match
   protected boolean isPrefixMatch = false;
 
+  private IMNodeIterator<N> currentChildrenIterator;
```

Comment:
Should this attribute be updated during traverse process? What about getting the current iterator via peeking the IteratorStack in AbstractTreeVisitor?